environment.yml

```diff
@@ -1,38 +1,53 @@
+# 1. Install a conda distribution.
+#    https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html
+# 2. Run: `mamba env create -f environment.yml`
+# 3. Activate: `conda activate spyglass`
+#
+# (lines intentionally left blank)
+#
+#
 name: spyglass
 channels:
   - conda-forge
   - defaults
+  # - pytorch # dlc-only
   - franklab
   - edeno
 dependencies:
-  - python>=3.9,<3.10
+  - bottleneck
+  # - cudatoolkit=11.3 # dlc-only
+  # - cupy # dlc-only
```

Comment:
cupy is for decoding as well. This can be removed.

Reply:
I understand this to mean 'remove the comment out, keep cupy for both versions' - is that right?

Reply:
Sorry, I meant remove cupy for both versions